src/rules/prefer-default-export.js

```diff
@@ -47,9 +47,16 @@ module.exports = {
         // if there are specifiers, node.declaration should be null
         if (!node.declaration) return
 
-        // don't count flow types exports
+        // don't warn on single type aliases or declarations
         if (node.exportKind === 'type') return
 
+        if (
+          node.declaration.type === 'TSTypeAliasDeclaration' ||
+          node.declaration.type === 'TypeAlias'
```

Comment:
@sharmilajesupaul @brieb should this also ignore interfaces? I believe the Babel/TypeScript AST type is `InterfaceDeclaration` and the typescript eslint parser type is `TSInterfaceDeclaration`.

Reply:
https://github.com/benmosher/eslint-plugin-import/pull/1425